deploy/docker-compose.yml

```diff
@@ -56,20 +56,24 @@ services:
     environment:
       - ZEN_ENV=${ZEN_WEB_ENV}
       - DONT_SYNC_SOURCEMAPS=true
-      - DATABASE_URL=${POSTGRES_DB_URI}
+      - DATABASE_URL=${DATABASE_URL}
       - HSTS=off
       - SSL_POLICY=Mozilla-Modern
       - VIRTUAL_HOST=${ZEN_WEB_HOST}
       - LETSENCRYPT_HOST=${ZEN_WEB_HOST}
       - LETSENCRYPT_EMAIL=${ZEN_WEB_EMAIL}
+      - VIRTUAL_ENV=/usr/local
```

Comment:
Why do we need a venv here?

Reply:
This is essentially a prod web instance

Reply:
I know right. Very frustrating. Our script do check all over the place, to see if you're running a venv, then kicks you out if you don't (by checking for VIRTUAL_ENV).
This is essentially a hack - the real fix is removing that from all the scripts.
It's a little "helper" code someone put in - but it's real dumb i.m.h.o.

For example:
```if [ -z "${VIRTUAL_ENV:-}" ] ; then
  echo 'You must be inside the Zenysis Virtual Environment to modify the Application Database.'
  exit 1
fi
```

It probably was extremely useful to someone at some point - but I don't think it's a good idea.

Alternative is pulling out all the code that checks if you're in a VENV? Probably a better idea?

Reply:
@Ideonvn after sleeping on this one - I'm just going to go ahead and remove the VIRTUAL_ENV check from scripts. I really don't think they belong there.

Reply:
Please do, imo this is an anti pattern to have it in prod. Still don't completely understand where it is needed?

Reply:
Agreed.

In theory one could argue that it could be relevant to first check the context in which the script is running.

i.m.h.o. it's not the responsibility of scripts/generate_migration_script.sh nor scripts/upgrade_database.sh to care. For the moment I'll allow the exception in the pipeline, where it's important to know if we're running cpython or pypy.